modules/ppcp-compat/src/CompatModule.php

```diff
@@ -79,6 +79,13 @@ public function run( ContainerInterface $c ): bool {
 			$this->initialize_wc_bookings_compat_layer( $logger );
 		}
 
+		add_action(
+			'woocommerce_paypal_payments_gateway_migrate',
+			function() {
+				delete_transient( 'ppcp_has_ppec_subscriptions' );
+			}
+		);
```

Comment:
This can be simplified and additionally add the `static` for the callback

```suggestion
add_action( 'woocommerce_paypal_payments_gateway_migrate', static fn() => delete_transient( 'ppcp_has_ppec_subscriptions' ) );
```